src/main/scala/com/atomist/rug/kind/core/ArtifactContainerMutableView.scala

```diff
@@ -1,24 +1,16 @@
 package com.atomist.rug.kind.core
 
-import com.atomist.rug.spi.{ExportFunction, ExportFunctionParameterDescription, MutableView, ViewSupport}
+import com.atomist.rug.spi._
 import com.atomist.source.{ArtifactContainer, DirectoryArtifact, FileArtifact}
 
-object ArtifactContainerMutableView {
```

Comment:
ah, these are unrelated fixes that I spotted during the work. These are changed to be uppercase and to come from the class names rather than hard-coding